examples/next-js/package.json

```diff
@@ -11,21 +11,21 @@
   "dependencies": {
     "@lens-protocol/react-web": "workspace:*",
     "@lens-protocol/wagmi": "workspace:*",
-    "react": "^18",
-    "react-dom": "^18",
     "next": "14.0.1",
-    "viem": "^1.16.6",
-    "wagmi": "^1.4.4"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "viem": "^1.19.5",
+    "wagmi": "^1.4.7"
   },
   "devDependencies": {
-    "typescript": "^5",
-    "@types/node": "^20",
-    "@types/react": "^18",
-    "@types/react-dom": "^18",
-    "autoprefixer": "^10.0.1",
-    "postcss": "^8",
-    "tailwindcss": "^3.3.0",
-    "eslint": "^8",
-    "eslint-config-next": "14.0.1"
+    "@types/node": "^18.18.11",
+    "@types/react": "^18.2.38",
+    "@types/react-dom": "^18.2.16",
+    "autoprefixer": "^10.4.16",
+    "eslint": "^8.54.0",
+    "eslint-config-next": "14.0.1",
+    "postcss": "^8.4.31",
+    "tailwindcss": "^3.3.5",
+    "typescript": "^5.3.2"
```

Comment:
this is all result of `pnpm -r update`

Reply:
I reckon we should stick to TS 5.2 for a little longer. 5.3 has only 2 weeks.